docs/sdk/java/user/create.mdx

```diff
@@ -49,7 +49,9 @@ userAttributes.put("age", Integer.valueOf(50));
 userAttributes.put("fav_color", "red");
 
 final List<UserRoleCreate> roleAssignments = Arrays.asList(
-                new UserRoleCreate("editor", "default")
+                new UserRoleCreate("editor", "default"),
+                new UserRoleCreate("moderator", "default"),
+                new UserRoleCreate("editor", "default", "myResourceType:myResourceInstance")
```

Comment:
Is this really allowed to pass 3 arguments here ?

Reply:
![Uploading Screenshot 2024-09-22 at 15.14.25.png…]()